public/scripts/main/misc/filters.js

```diff
@@ -1,87 +0,0 @@
-// this module handles front end filters which are piped through "|" bars between double brackets
```

Comment:
another file deleted - this makes me nervous for the stability of you build - was anything else tampered?